app/main.py

```diff
@@ -1,2 +1,45 @@
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 2 / 3
+    ) -> None:
+        self._size = 0
+        self._capacity = initial_capacity
+        self._load_factor = load_factor
+        self._table = [[] for _ in range(self._capacity)]
+
+    def __setitem__(self, key: str, value: int) -> None:
```

Comment:
Indicate that `key` should be `Hashable` in the type annotation and that value could be `Any`, not only `int`. Apply this change in other places as well